keyrings/alt/pyfs.py

```diff
@@ -9,14 +9,44 @@
 from keyring.backend import KeyringBackend, NullCrypter
 from . import keyczar
 
+# support getting module version
+import pkg_resources
```

Comment:
pkg_resources is discouraged for this use. Let's use importlib.metadata instead. Regardless, you need to declare the dependency.